integration/tracing/tracing-opentelemetry/pom.xml

```diff
@@ -46,11 +46,6 @@
             <artifactId>jakarta.xml.ws-api</artifactId>
             <optional>true</optional>
         </dependency>
-        <dependency>
```

Comment:
I would not move dependencies for traceability. Event though it is a best practice.